doc/commands.md

```diff
@@ -128,48 +128,49 @@ Rewriting rules for definable symbols are declared using the `rule` command.
 ```
 rule add zero      $n ↪ $n
 rule add (succ $n) $m ↪ succ (add $n $m)
+rule mul zero      _  ↪ zero
 ```
 
-Note that rewriting rules can also be defined simultaneously, using the `with`
-keyword instead of the `rule` keyword for all but the first rule.
+Terms prefixed by the sigil `$` and `_` are pattern variables.
+
+**Higher-order pattern-matching**.
+Lambdapi allows higher-order pattern-matching following Miller’s patterns.
```

Comment:
Lambdapi allows higher-order pattern-matching on patterns à la Miller but modulo β-equivalence only (and not βη).